etna/datasets/tsdataset.py

```diff
@@ -416,10 +417,26 @@ def inverse_transform(self, transforms: Sequence["Transform"]):
         Applied in reversed order.
         """
         # TODO: return regressors after inverse_transform
+        inverse_transform_target_components_flg = self.target_components is not None
+        target_df, target_components_df = None, None
+        if inverse_transform_target_components_flg:
+            target_df = self.to_pandas(features=["target"])
+            target_components_df = self.get_target_components()
+            self.drop_target_components()
+
         for transform in reversed(transforms):
             tslogger.log(f"Inverse transform {repr(transform)} is applied to dataset")
             transform.inverse_transform(self)
 
+        if inverse_transform_target_components_flg:
+            inverse_transformed_target_df = self.to_pandas(features=["target"])
+            inverse_transformed_target_components_df = inverse_transform_target_components(
```

Comment:
If this is so similar to code in transforms, can we expand `inverse_transform_target_components` somehow?